apps/api/test/fixture/reports.ts

```diff
@@ -26,3 +27,10 @@ export const addMilestone = {
   notes: 'Test Note 1',
   reason: '',
 };
+
+// Report 4 data
+export interface ReportFourItem {
```

Comment:
A fixture is supposed to provide test data. We'd better define returning object types for report endpoints.

Reply:
Yeah, I wasn't sure where to put that. Just so I understand, are you suggesting I make a new file for report types?

Reply:
You can find an example in `api/src/report/types`.